src/lib/GUID.cpp

```diff
@@ -23,13 +25,57 @@
 namespace libone
 {
 
+// {FFFFFFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFF}
+enum { MaxStringGUIDLength = 38};
+// FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+enum { MinStringGUIDLength = 32};
```

Comment:
These two enums look like they could be made private to the GUID class, as the only time they're used are in GUID::from_string() (you can look at the `static const` private members in FileNode.h for inspiration)

Reply:
I see what your are meaning. I think i copied without much thought from Qt's Quuid class...
Will be fixed.

Reply:
After a brief search, I saw that single-member enums are used to get free-standing constants to work around older cpp compilers.. I guess Qt could be kind of in such a case, as it can sometimes target some unconventional architectures and compilers (embedded systems with years-old toolchains..). So I can see why they did it like that!

Reply:
Thanks for sharing that information! This wasn't clear to me.

I replaced the enums anyway, since we can safely assume reasonably modern compilers.